README.md

```diff
@@ -785,16 +785,24 @@ Models are represented as a typealias `typealias Model = String`.
 
 ```swift
 public extension Model {
-		static let gpt4_1106_preview = "gpt-4-1106-preview"
-		static let gpt4_vision_preview = "gpt-4-vision-preview"
+    static let gpt4_turbo_preview = "gpt-4-turbo-preview"
+    static let gpt4_vision_preview = "gpt-4-vision-preview"
+    static let gpt4_0125_preview = "gpt-4-0125-preview"
+    static let gpt4_1106_preview = "gpt-4-1106-preview"
     static let gpt4 = "gpt-4"
+    static let gpt4_0613 = "gpt-4-0613"
     static let gpt4_0314 = "gpt-4-0314"
     static let gpt4_32k = "gpt-4-32k"
+    static let gpt4_32k_0613 = "gpt-4-32k-0613"
     static let gpt4_32k_0314 = "gpt-4-32k-0314"
+    
     static let gpt3_5Turbo = "gpt-3.5-turbo"
+    static let gpt3_5Turbo_0125 = "gpt-3.5-turbo-0125"
     static let gpt3_5Turbo_1106 = "gpt-3.5-turbo-1106"
+    static let gpt3_5Turbo0613 = "gpt-3.5-turbo-0613"
     static let gpt3_5Turbo0301 = "gpt-3.5-turbo-0301"
```

Comment:
Perhaps this should be `gpt3_5Turbo_0613` like the others (same for 0301), but not a deal breaker 👍

Reply:
Good catch @SunburstEnzo, code is adjusted! 🚀